packages/nextjs/app/builders/0xee3869d7dfc3e1a2cb6399e00c6a39e019e1a80b/page.tsx

```diff
@@ -0,0 +1,67 @@
+import Link from "next/link";
+import { Address } from "@scaffold-ui/components";
+import { Globe, Linkedin } from "lucide-react";
+import type { NextPage } from "next";
+import { BlockieAvatar } from "~~/components/scaffold-eth";
+
+const OscarYuBuilderPage: NextPage = () => {
+  const builderAddress = "0xEe3869d7dFc3e1a2CB6399E00C6a39E019E1A80b";
+  const linkedinUrl = "https://www.linkedin.com/in/oscar-yu-cheuk-chun/";
+  const websiteUrl = "https://oscar-yu-portfolio.vercel.app/";
```

Comment:
Since these are all constant, it makes sense to place them outside of the component. So they're created only once at module initialization, rather than every render.